src/openrct2/park/ParkFile.cpp

```diff
@@ -1516,9 +1516,9 @@ namespace OpenRCT2
                         return true;
                     });
                     cs.Ignore<uint64_t>();
-                    cs.Ignore<ride_id_t>();
-                    cs.Ignore<ride_id_t>();
-                    cs.Ignore<ride_id_t>();
+                    cs.Ignore<RideId>();
```

Comment:
I wonder if these should perhaps become `uint16_t`'s just thinking about in a future where RideId is not `uint16_t`

Reply:
I wonder, currently we have rides capped at 1000, so that makes only 1.5% of used space. I wouldn't mind a static assert so we don't forget when we happen to change it.

Reply:
I think its better to handle this when the time comes to, then we simply use the park version.